### DISCUSSION TITLE
Adding opts to default installed plugins

### DISCUSSION THREAD

**Discussion by enilsen16** [LazyVim/LazyVim#340] [Q&A] [answered]
Having trouble extending neotree to included the opts to show all files rather than hiding dot and gitignored files.

In lua/plugins/neotree.lua:
```
return {
  {
    "nvim-neo-tree/neo-tree.nvim",
    opts = function(_, opts)
      vim.list_extend(opts.filesystem, {
        filtered_items = {
          visible = true,
          hide_dotfiles = false,
          hide_gitignored = false,
        },
      })
    end,
  },
}
```
Not quite sure if I am not extending opts correctly or if there's something else I am missing.

**Comment by dpetka2001** [comment, marked as answer]
Please see [here](https://github.com/LazyVim/LazyVim/discussions/334). It's the same problem.
